src/stratigraphy/util/find_depth_columns.py

```diff
@@ -38,26 +32,51 @@ def value_as_float(string_value: str) -> float:  # noqa: D103
                 entries.append(DepthColumnEntry(word.rect, value))
             elif include_splits:
                 # support for e.g. "1.10-1.60m" extracted as a single word
-                regex2 = re.compile(r"^-?([0-9]+(\.[0-9]+)?)[müMN\\.]*\W+([0-9]+(\.[0-9]+)?)[müMN\\.]*$")
-                match2 = regex2.match(input_string)
-
-                if match2:
-                    value1 = value_as_float(match2.group(1))
-                    first_half_rect = fitz.Rect(
-                        word.rect.x0, word.rect.y0, word.rect.x1 - word.rect.width / 2, word.rect.y1
-                    )
-                    entries.append(DepthColumnEntry(first_half_rect, value1))
-
-                    value2 = value_as_float(match2.group(3))
-                    second_half_rect = fitz.Rect(
-                        word.rect.x0 + word.rect.width / 2, word.rect.y0, word.rect.x1, word.rect.y1
-                    )
-                    entries.append(DepthColumnEntry(second_half_rect, value2))
+                entries.extend(extract_layer_depth_interval_entries(input_string, word.rect))
         except ValueError:
             pass
     return entries
 
 
+def value_as_float(string_value: str) -> float:  # noqa: D103
+    # OCR sometimes tends to miss the decimal comma
+    parsed_text = re.sub(r"^-?([0-9]+)([0-9]{2})", r"\1.\2", string_value)
+    return abs(float(parsed_text))
+
+
+def extract_layer_depth_interval_entries(
+    text: str, rect: fitz.Rect, require_start_of_string: bool = True
+) -> list[DepthColumnEntry]:
```

Comment:
Wouldn't it be more natural to return `LayerDepthColumnEntry | None` here, as we always either return an empty list, or a list with two entries (a start and an end) which is exactly what LayerDepthColumnEntry  represents.